eth/config.go

```diff
@@ -166,6 +166,6 @@ type Config struct {
 
 	// MuirGlacier block override (TODO: remove after the fork)
 	OverrideMuirGlacier *big.Int `toml:",omitempty"`
-	
+	// Added by EE
```

Comment:
The linter also complained about this line being blank.

Reply:
I think this is safe to delete... not sure why that blank was added 😬